addons/mail/static/src/core/web/discuss_patch.xml

```diff
@@ -7,5 +7,8 @@
         <xpath expr="//*[@t-ref='root']" position="inside">
             <MessagingMenu t-if="ui.isSmall"/>
         </xpath>
+        <xpath expr="//*[@t-ref='root']" position="before">
+            <ControlPanel t-if="!ui.isSmall"/>
+        </xpath>
```

Comment:
ok for now but the whole mobile experience / breadcrumb / etc is currently far from optimal (or even kind of broken)